src/slices/authSlice.js

```diff
@@ -31,3 +31,4 @@ export const selectUserCredentials = state => {
     authToken: state.auth.user.auth_token
   };
 };
+export const selectIsAuth = state => state.auth.user.isAuth;
```

Comment:
Need to put it in a file selectors.js (index maybe) 
isAuth = useSelector() and export isAuth

Reply:
I like concept that all things considering state slices pass in one file. Here best practice from redux itself:
https://redux.js.org/tutorials/essentials/part-5-async-logic

![image](https://user-images.githubusercontent.com/51961835/176392246-bc3ffccc-fbf7-49fa-b705-f77f8ac9898e.png)